### PR TITLE
Implement onChain payment send mutations

### PR DESCRIPTION
**Pull request by samerbuna**

### CODE REVIEW

File: src/servers/graphql-main-server.ts [hunk 1]
```diff
@@ -34,6 +34,8 @@ export async function startApolloServerForCoreSchema() {
         lnNoAmountInvoicePaymentSend: isAuthenticated,
 
         intraLedgerPaymentSend: isAuthenticated,
+
+        onChainPaymentSendAll: isAuthenticated,
```
Comment (dolcalmi) [inline src/servers/graphql-main-server.ts:38]: are we missing `onChainPaymentSend` ?

File: src/graphql/types/scalar/on-chain-address.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ const OnChainAddress = new GT.Scalar({
 
 function validOnChainAddressValue(value) {
   // TODO: verify/improve. Use bc1/tb1 prefixes?
-  if (value.match(/^[A-Fa-f0-9]+$/i)) {
+  if (value.match(/^[A-Z0-9]+$/i)) {
```
Comment (dolcalmi) [inline src/graphql/types/scalar/on-chain-address.ts:20]: did you test this? maybe we should have a helper with some unit tests

**Review by dolcalmi — CHANGES_REQUESTED** [review]

File: src/graphql/types/scalar/on-chain-address.ts (hunk 2 above)
Comment (bodymindarts) [reply src/graphql/types/scalar/on-chain-address.ts:20]: We cannot constrain btc addresses to only contain uppercase. Lower case addresses are not only valid but the norm for most wallets.

Comment (samerbuna) [reply src/graphql/types/scalar/on-chain-address.ts:20]: This the current check we have. It's case insensitive.

File: src/servers/graphql-main-server.ts (hunk 1 above)
Comment (samerbuna) [reply src/servers/graphql-main-server.ts:39]: Yep. Fixed.

**Review by dolcalmi — DISMISSED** [review]

File: src/graphql/types/scalar/on-chain-address.ts (hunk 2 above)
Comment (bodymindarts) [reply src/graphql/types/scalar/on-chain-address.ts:20]: Ah right `/i` -> case insensitive.

**Review by bodymindarts — APPROVED** [review]

**Review by vindard — APPROVED** [review]

**Comment by vindard** [comment]
@samerbuna looks like this needs a rebase

**Comment by bodymindarts** [comment]
> @samerbuna looks like this needs a rebase
taken care of - can you re-approve @vindard

**Review by bodymindarts — APPROVED** [review]

**Review by vindard — APPROVED** [review]

**Pull request merged**